packages/holocron/src/updateModuleRegistry.js

```diff
@@ -18,44 +18,44 @@ import loadModule from './loadModule.node';
 
 export function areModuleEntriesEqual(firstModuleEntry, secondModuleEntry) {
   return !!(
-    firstModuleEntry &&
-    secondModuleEntry &&
-    shallowEqual(firstModuleEntry.browser, secondModuleEntry.browser) &&
-    shallowEqual(firstModuleEntry.legacyBrowser, secondModuleEntry.legacyBrowser) &&
-    shallowEqual(firstModuleEntry.node, secondModuleEntry.node)
+    firstModuleEntry
+    && secondModuleEntry
+    && shallowEqual(firstModuleEntry.browser, secondModuleEntry.browser)
+    && shallowEqual(firstModuleEntry.legacyBrowser, secondModuleEntry.legacyBrowser)
+    && shallowEqual(firstModuleEntry.node, secondModuleEntry.node)
   );
 }
 
 function defaultGetModulesToUpdate(curr, next) {
   return Object
     .keys(next)
-    .filter(moduleName => !areModuleEntriesEqual(curr[moduleName], next[moduleName]));
+    .filter((moduleName) => !areModuleEntriesEqual(curr[moduleName], next[moduleName]));
 }
 
 export default async function updateModuleRegistry({
   moduleMap: newModuleMap,
   onModuleLoad = () => null,
-  batchModulesToUpdate = x => [x],
+  batchModulesToUpdate = (x) => [x],
   getModulesToUpdate = defaultGetModulesToUpdate,
 }) {
   const currentModuleMap = getModuleMap().toJS();
-  const modulesToUpdate =
-    batchModulesToUpdate(getModulesToUpdate(currentModuleMap.modules || {}, newModuleMap.modules));
+  const modulesToUpdate = batchModulesToUpdate(
+    getModulesToUpdate(currentModuleMap.modules || {}, newModuleMap.modules));
```

Comment:
this formatting pattern seems to be all over, are you running something like prettier on save ? if so i recommend disabling this rule.

Reply:
disabling this, would be reverting this changes

Reply:
sorry, i meant in prettier